### PR TITLE
Get all DAOs for a member

### PR DESCRIPTION
**Pull request by n13**
## Purpose - 

Provides a service calls to fetch all DAOs an account is part of 

## Notes

We should store the network along with the account for all accounts, so we always know which network to use. 

We can start assuming "telos" for any accounts we didn't store this data as a default. 

## Sample code to display the IPFS as an image

The logo comes as IPFS hash, like this:
Settings Logo: QmbyW3yY7jTjUi6XCkSy5GMsPDtonXBQ6e5nTFj3Dh6P9z:jpeg

To show this logo, we can use ipfs_http_client

```
dependencies:
  flutter:
    sdk: flutter
  http: ^0.13.4
  ipfs_http_client: ^0.1.0
  flutter_image: ^1.1.0


=======================

import 'package:http/http.dart' as http;
import 'package:ipfs_http_client/ipfs_http_client.dart';

// IPFS hash from your response
final ipfsHash = 'QmbyW3yY7jTjUi6XCkSy5GMsPDtonXBQ6e5nTFj3Dh6P9z';

// Create an instance of IPFS client
final ipfsClient = IPFSHttpClient(http.Client());

// Fetch the image data from IPFS
final imageData = await ipfsClient.cat(ipfsHash);

==================================

import 'package:flutter/material.dart';
import 'package:flutter_image/flutter_image.dart';

// Create a Widget to display the image
final imageWidget = Image.memory(imageData);

// Display the image in your Flutter ImageView
return Scaffold(
  body: Center(
    child: imageWidget,
  ),
);



```

### CODE REVIEW

File: lib/core/network/models/dao_data_model.dart [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    required this.settingsDaoTitle,
+    required this.settingsLogo,
+    required this.settingsDaoUrl,
+  });
+
+  factory DaoData.fromJson(Map<String, dynamic> json) {
+    final Map<String, dynamic> settings = json['settings'][0];
+    return DaoData(
+      docId: json['docId'],
+      detailsDaoName: json['details_daoName_n'],
```
Comment (gguijarro-c-chwy) [inline lib/core/network/models/dao_data_model.dart:20]: who named these params??...   details_daoName_n 
Camel + snake what a mess lol

Comment (n13) [reply lib/core/network/models/dao_data_model.dart:20]: Lol - you are right, I will create an update.

Comment (n13) [reply lib/core/network/models/dao_data_model.dart:20]: It was ChatGPT...

Comment (n13) [reply lib/core/network/models/dao_data_model.dart:20]: Actually - wait on our end the names are cool - the AI did the right thing

We can't change the names we get from the service obviously... they are messy.

**Pull request merged**